extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js

```diff
@@ -77,8 +63,12 @@ define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
 				lookups.extendSlotTargeting(slotName, slotTargeting);
 			}
 
-			gptHelper.pushAd(slotName, slotPath, slotTargeting, doSuccess, doHop);
-			flushIfNeeded();
+			if (gptSraHelper && extra.sraEnabled) {
```

Comment:
1. While it is useless for mobile providers, `//extensions/wikia/AdEngine/js/GptSraHelper.js` dependency is added only to **adengine2_desktop_js** group

2. Doc added